searx/botdetection/limiter.py

```diff
@@ -51,7 +51,6 @@
     http_accept,
     http_accept_encoding,
     http_accept_language,
-    http_connection,
     http_user_agent,
     ip_limit,
     ip_lists,
```

Comment:
> I have seen many many bots sending the connection close header on my server. And this rule have blocked a lot of bots.

After https://github.com/searxng/searxng/pull/2894 has been reviewed & merged we can consider to make this rules configurable in the `limiter.toml`.